### ISSUE TITLE
Allow passing `--find-links` to pip install

### ISSUE THREAD

**Issue by aersam** [microsoft/Oryx#1891]
### Feature Request

Hi there! I always pass a requirements.txt file to Oryx and it builds my stuff. However it would be nice if you could provide the wheel's myself by passing the value of the `--find-links` Parameter of `pip`. I think an environment variable would be needed for this

**Comment by aersam** [comment]
No, because I dont want to deal with the authentication there. Delivering the wheels would be easier

**Comment by pauld-msft** [comment]
@aersam Yes, we would be more than happy to accept a PR. 

I do want to give you fair warning that fully integrating any changes with our Azure partners and rolling them out takes some time (up to months). Regardless, the sooner that we get something merged in here, the sooner that it will eventually be integrated!
